gCoffee/app/src/test/java/com/mborowiec/gcoffee/OrderTest.java

```diff
@@ -44,4 +44,64 @@ public void testCheckMenuNull() {
         boolean test = testClass.menuContainsItem(null, coffeeList);
         assertFalse(test);
     }
+
+    @Test
+    public void testFindMatchExactMatch() {
+        String testString = testClass.findMatch("latte", coffeeList);
+        assertEquals("latte", testString);
+    }
+
+    @Test
+    public void testFindMatchSimpleTypo() {
```

Comment:
What about giving more descriptive test names?.. e.g. "whitespaceMissing", "middleLetterMissing", "lastLetterMissing" etc. From my experience, it is helpful if some tests start failing :)